internal/server/audit/logfile/logfile.go

```diff
@@ -0,0 +1,63 @@
+package logfile
+
+import (
+	"encoding/json"
+	"os"
+	"sync"
+
+	"github.com/hashicorp/go-multierror"
+	"go.flipt.io/flipt/internal/server/audit"
+	"go.uber.org/zap"
+)
+
+const sinkType = "logfile"
+
+// LogFileSink is the structure in charge of sending Audits to a specified file location.
+type LogFileSink struct {
+	logger *zap.Logger
+	file   *os.File
+	mtx    sync.Mutex
+	enc    *json.Encoder
+}
+
+// NewSink is the constructor for a LogFileSink.
+func NewSink(logger *zap.Logger, path string) (audit.Sink, error) {
+	file, err := os.OpenFile(path, os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0666)
+	if err != nil {
+		return nil, err
```

Comment:
should we maybe wrap this like:

```suggestion
		return nil, fmt.Errorf("opening log file: %w", err)
```